pkgs/by-name/pa/panvimdoc/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  pandoc,
+  vim,
+  neovim-unwrapped,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+
+  pname = "panvimdoc";
+  version = "4.0.1";
+
+  src = fetchFromGitHub {
+    owner = "kdheepak";
+    repo = "panvimdoc";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-HmEBPkNELHC7Xy0v730sQWZyPPwFdIBUcELzNtrWwzQ=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  buildPhase = ''
+    install -Dm444 scripts/* -t $out/share/scripts
+    install -Dm444 lib/* -t $out/share/lib
+    install -Dm755 panvimdoc.sh -t $out/share
+    mkdir -p $out/bin
+    makeWrapper $out/share/panvimdoc.sh $out/bin/panvimdoc --prefix PATH : ${
+      lib.makeBinPath [
+        pandoc
+        vim
+        neovim-unwrapped
+      ]
+    }
+  '';
+
+  meta = {
```

Comment:
```suggestion
  '';
  
  passthru = {
    tests.version = testers.testVersion { package = panvimdoc; };
    updateScript = nix-update-script { };
  };

  meta = {
```

Reply:
Unfortunately, the script doesn't have a `--version` or `-v` flag (and doesn't output the version).

Reply:
Oh ok, just ignore this remark then.